js/index.js

```diff
@@ -0,0 +1,56 @@
+//copyright vars
+const today = new Date();
+const thisYear = today.getFullYear();
+const footer = document.querySelector("#footer");
+const copyright = document.createElement("p");
+
+//copyright logo
+const logoCopyright = document.createElement("p");
+logoCopyright.innerHTML = "&#9881";
+logoCopyright.style.display = "inline";
+footer.appendChild(logoCopyright);
+
+copyright.innerHTML = "Cristian Moran" + "- " + thisYear;
+copyright.style.display = "inline";
+footer.appendChild(copyright);
+
+var headings = document.querySelectorAll("h2");
+headings.forEach(function (heading) {
+  heading.style.color = "#000080";
+});
+
+//this is a variable refering to the buttonContainer
+var container = document.querySelector("nav button");
```

Comment:
How about just changing the variable name to `buttonContainer` and removing the comment? That would be a good opportunity to utilize "self-documenting" code, which is good at making this kind of comment less necessary!